.github/workflows/ci.yml

```diff
@@ -132,12 +142,13 @@ jobs:
         run: |
           sudo apt-add-repository -y "ppa:ubuntu-toolchain-r/test"
           sudo apt update
-          sudo apt install -y gcc-4.8 g++-4.8 bison flex uuid-dev openssl libcurl4-openssl-dev ccache libpython3-dev liblua5.1-0-dev libssh2-1-dev
+          sudo apt install -y gcc-4.8 g++-4.8 bison flex uuid-dev openssl libcurl4-openssl-dev ccache libpython3-dev liblua5.1-0-dev libssh2-1-dev libsqliteodbc
+          sudo ln -s /usr/lib/x86_64-linux-gnu/odbc/libsqlite3odbc.so /usr/lib/x86_64-linux-gnu/libsqlite3odbc.so
```

Comment:
I feel like this is a problem with the `libsqliteodbc` library, but you are right that it introduces quite a friction and cryptic errors, we could maybe print some warnings?

Reply:
Even if the `libsqliteodbc` package is broken, we want `bootstrap.sh` followed by `make` to produce a working binary without any extra steps like symlinking.  So I would say we either need to figure out how to get `soci` to work without the symlink, or put the `ln -s` command in (a script called from) bootstrap.sh.

Reply:
the binary will be working fine, the symlink is only (apparently) needed on ubuntu for the ODBCService iff we don't want to employ an odbc.ini file, and without an odbc.ini file the connection string is inherently platform specific anyway

Reply:
The binary does not work on my Ubuntu 18.04 without the symlink.  With the symlink, it works fine.

Reply:
so the agent doesn't even start up?

Reply:
if I uninstall sqliteodbc on mac, the agent starts up just fine, but of course the sql processors won't work
```
[org::apache::nifi::minifi::processors::ExecuteSQL] [error] SQLProcessor: 'Error connecting to database: [iODBC][Driver Manager]dlopen(/usr/local/lib/libsqlite3odbc.dylib, 6): image not found (SQL state 00000)'
```

Reply:
Without the symlink, my agent starts up, but I get this error in PutSQL::onTrigger:
```
[org::apache::nifi::minifi::processors::PutSQL] [error] SQLProcessor: 'Error connecting to database: [iODBC][Driver Manager]libsqlite3odbc.so: cannot open shared object file: No such file or directory (SQL state 00000)'
```

Reply:
But it does work with the connection string `Driver=/usr/lib/x86_64-linux-gnu/odbc/libsqlite3odbc.so;Database=/tmp/my_database.db`, so I guess that's OK.

It would be nice if the SQL processors could use the driver and database definitions in `odbcinst.ini` and `odbc.ini`, respectively, but that can be an improvement we'll add later.

Reply:
created a ticket to track this issue: https://issues.apache.org/jira/browse/MINIFICPP-1534